doc/source/configuration/magnum-capi.rst

```diff
@@ -1,45 +1,127 @@
+
+*Note* - One important distinction between azimuth-config and stackhpc-kayobe-config is that the environments in azimuth-config are `layered`. This can be seen in the ``ansible.cfg`` file for each environment, which will contain a line such as ``inventory = ../base/inventory,../ha/inventory,../capi-mgmt/inventory,./inventory`` showing the inheritance chain for variables defined in each environment. See `these docs <https://stackhpc.github.io/azimuth-config/environments/>`_ for more details.
+
+In addition to setting the required infrastructure variables, Terraform must also be configured to use a remote state store (either GitLab or S3) for the seed VM state. To do so, follow the instructions found `here <https://stackhpc.github.io/azimuth-config/repository/terraform/>`_.
+
+The HA cluster also contains a deployment of `kube-prometheus-stack <https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack>`_ for monitoring and alerting. To send the cluster alerts to Slack, the ``alertmanager_config_slack_webhook_url`` variable should be set in ``environments/<site-specific-name>/inventory/group-vars/all/secrets.yml``. If the repository was encrypted correctly above, this file will automatically be encrypted before a git push. Run ``git-crypt status -e`` to verify that this file is included in the encrypted list before git-committing the webhook URL.
+
+The final step before beginning deployment of the CAPI management cluster is to provide some cloud credentials. It is recommended that the CAPI management cluster is deployed in an isolated OpenStack project. After creating the target project (preferably using openstack-config), generate an application credential for the project using the Identity tab in Horizon and then download the corresponding ``clouds.yaml`` file and place it in ``environments/<site-specific-name>/clouds.yaml``.
+
+To deploy the CAPI management cluster using this site-specific environment, run
+
+.. code-block:: bash
+
+    # Activate the environment
+    ./bin/activate <site-specific-name>
+
+    # Install or update the local Ansible Python venv
+    ./bin/ensure-venv
+
+    # Install or update Ansible dependencies
+    ansible-galaxy install -f -r ./requirements.yml
+
+    # Run the provision playbook from the azimuth-ops collection
+    # NOTE: THIS COMMAND RUNS A DIFFERENT PLAYBOOK FROM
+    # THE STANDARD AZIMUTH DEPLOYMENT INSTRUCTIONS
+    ansible-playbook stackhpc.azimuth_ops.provision_capi_mgmt
```

Comment:
Does this not need a .yml extension? I might just not be familiar with the syntax here